src/verify/VerificationEducationScreen.test.tsx

```diff
@@ -286,4 +286,33 @@ describe('VerificationEducationScreen', () => {
     fireEvent.changeText(getByTestId('PhoneNumberField'), '51231234')
     expect(getByTestId('VerificationEducationContinue')).not.toBeDisabled()
   })
+
+  it("continue button enabled with Côte d'Ivoire number", () => {
+    const store = createMockStore({
+      account: {
+        e164PhoneNumber: '2123456789',
+        defaultCountryCode: '+255',
+      },
+      stableToken: {
+        balances: { [Currency.Dollar]: '0' },
+      },
+      verify: {
+        currentState: idle(),
+        status: { numAttestationsRemaining: 3 },
+        actionableAttestations: [],
+        komenciAvailable: KomenciAvailable.Yes,
```

Comment:
i think out of all this mocked state, the `komenciAvailable` one is the only one this test depends on right? not sure if the stable token/cached phone number etc are necessary here

Reply:
i'm a bit out of the loop with what happened, but how is this test different to the one above for `continue button disabled when invalid number` (since here we are essentially testing that the number format is correct before allowing the user to continue)? we don't have any specific code for cote d'ivoire so am confused why a specific test is needed. would be nice to leave a comment if we do need it :)

Reply:
Can add a comment - it's similar to the above 'continue button disabled when invalid number', but tests that same logic with a Côte d'Ivoire number. I wanted to make sure that this screen would work with the new number format.

Reply:
@kathaypacific - was able to remove this test entirely in https://github.com/valora-inc/wallet/pull/2473/commits/50decd65d418d03b6e11e726884a146750bb2bc1 after merging #2478!